integrations/launchdarkly/src/components.tsx

```diff
@@ -86,39 +86,65 @@ export const configBlock = createComponent<
         return (
             <configuration>
                 {isAdaptiveContentEnabled ? (
-                    <block>
-                        <input
-                            label="Secret key"
-                            hint={
-                                <text>
-                                    The secret key from your LaunchDarkly{' '}
-                                    <link
-                                        target={{
-                                            url: 'https://app.LaunchDarkly.co/envs/current/settings/app-environments',
+                    <box>
+                        <vstack>
```

Comment:
We need the same `vstack` in `Bucket` too

Reply:
Going to make the change in another PR and merge this for testing.